### ISSUE TITLE
[Bug]: Spelling mistake in json when fetching LNURLw from Bolt card

### ISSUE THREAD

**Issue by Amperstrand** [btcpayserver/btcpayserver#5440]
### What is your BTCPay version?

v1.11.7+a921504bc

### How did you deploy BTCPay Server?

Docker

### What happened?

When using a Boltcard created by BTCPayserver to check out using NFC in the following apps, I get errors:

Wallet of Satoshi: "Problem processing the LNURL"

BTCPayserver POS checkout on Android device: "Could not fetch info from LNURL-Withdraw: A tag identifying the LNURL endpoint was not received.

The boltcard read with an NFC reader app gives the following URL:

lnurlw://my.btcpayserv.com/plugins/lnbank/api/boltcard/pay?p=6602F0121FC3063C09D3073A2F57D339&c=8FA22212D74417BE

When accessing this url in a browser and replacing lnurlw with https, I get the following json:

```
{"result":{"callback":"https://my.btcpayserv.com/plugins/lnbank/api/boltcard/pay-callback","k1":"5d33882b-154a-44e1-9738-16d5f2f3a46b","tag":"withdrawRequest","defaultDescription":"boltcard1","minWithdrawable":1000,"maxWithdrawable":997000,"currentBalance ":997000,"balanceCheck":null},"id":30,"exception":null,"status":5,"isCanceled":false,"isCompleted":true,"isCompletedSuccessfully":true,"creationOptions":0,"asyncState":null,"isFaulted":false}
```

In the above json, there is a linebreak or space at "currentBalance ". Perhaps this is what is causing the issue.

For reference, here is json from an LNBits boltcard which works:

```
{"tag":"withdrawRequest","callback":"https://legend.lnbits.com/boltcards/api/v1/lnurl/cb/F6b9FnN77NQvSfoKgWi4Px","k1":"F6b9FnN77NQvSfoKgWi4Px","minWithdrawable":1000,"maxWithdrawable":50000000,"defaultDescription":"Boltcard (refund address lnurl://LNURL1DP68GURN8GHJ7MR9VAJKUEPWD3HXY6T5WVHXXMMD9A3X7MR5VDSHYERN9ASHQ6F0WCCJ7MRWW4EXCUP0GCMXYW2XDE8RWD6W29M9XEN0FDN4W6F52PUQT3UFAA)","payLink":"lnurlp://legend.lnbits.com/boltcards/api/v1/lnurlp/F6b9FnN77NQvSfoKgWi4Px"}
```

### How did you encounter this bug?

1. Add LNBank
2. Create Withdraw configuration with static LNURLw link to make sure the LNBank works --> it does
3. Create Withdraw confirguration for the bolt card
4. Write the boltcard with Bolt Card Writer android app
5. Use boltcard to pay wallet of satoshi or check out in btcpayserver POS.

### What browser do you use?

BTCPayserver POS mode is running in Chrome on Android. It works with boltcard issued by LNBits but not BTCPayserver.

### Are you sure this is a bug report?

- [X] I confirm this is a bug report

**Comment by dennisreimann** [comment]
I fixed the space typo in Kukks/LNURL#9 and think this issue here is that the result is wrapped inside a the result key (`{ "result": LNURLW }`). Will take a look and fix it, thanks for reporting!

**Comment by dennisreimann** [comment]
Should be solved in LNbank v1.8.1 — can you verify it works for you?

**Comment by Amperstrand** [comment]
> Should be solved in LNbank v1.8.1 — can you verify it works for you?

Tested and looks like it works. Bolt Cards go Brrr again. Will report back if I run into further issues.
